controller/endpoints.go

```diff
@@ -80,7 +80,7 @@ func (c *AWSLBReadvertiserController) createTwoWayEndpointMergePatch(endpoint *c
 		return nil, fmt.Errorf("failed to patch bytes")
 	}
 
-	_, err = c.client.CoreV1().Endpoints(metav1.NamespaceDefault).Patch(endpoint.Name, types.StrategicMergePatchType, patchBytes)
+	_, err = c.client.CoreV1().Endpoints(metav1.NamespaceDefault).Patch(context.TODO(), endpoint.Name, types.StrategicMergePatchType, patchBytes, metav1.PatchOptions{})
```

Comment:
Reviewers please provide feedback about the right usage of the context

Reply:
```suggestion
	_, err = c.client.CoreV1().Endpoints(metav1.NamespaceDefault).Patch(ctx, endpoint.Name, types.StrategicMergePatchType, patchBytes, metav1.PatchOptions{})
```